src/pages/content/index.vue

```diff
@@ -26,7 +26,7 @@ import Pagination from "@components/Pagination";
 export default {
 	head() {
 		return {
-			title: "LernStore",
+			title: "Lernstore",
```

Comment:
```suggestion
			title: "LernStore",
```
To be 100% correct, it's called `Lern-Store`, but we should change this in a seperate pull request

Reply:
not even sure why that is changed. I think that was just a mistake. sorry for the extra work